Jenkinsfile

```diff
@@ -26,7 +26,7 @@
 
 properties([
 	parameters([
-		booleanParam(name: 'deploy_prod_from_non_master', defaultValue: false, description: 'Whether to deploy to prod-like envs for builds of this project\'s non-master branches.'),
+		booleanParam(name: 'deploy_prod_from_non_master', defaultValue: true, description: 'Whether to deploy to prod-like envs for builds of this project\'s non-master branches.'),
```

Comment:
This meets the simple AC for defaults of BFD-3077, but I think this may have been in error. There are some scenarios that have engineers using the primary build/deliver/deployment pipeline to evaluate relevant changes to  use the primary deployment pipeline to artifact generation or the build/delivery/deployment job itself–these kinds of changes might be best evaluated with this specific job. This could certainly lead to inadvertent deployments of half-baked changes to production...

What we had discussed was only defaulting to deployments from non master when executed from a well-qualified, production-ready tag, e.g. vaguely semver'd `X.Y.Z` or similar **and** prohibiting deployments beyond `test` from branches including, including `master`.

Reply:
It sounds to me like that will require some re-thought on how to implement this. Should I back out that change so we can get the rest into master?